Exec/science/flame_wave/analysis/vis_3d/andes-slice.submit

```diff
@@ -2,15 +2,16 @@
 #SBATCH -A ast106
 #SBATCH -J plots
 #SBATCH -N 1
-#SBATCH -t 2:00:00
+#SBATCH -t 8:00:00
```

Comment:
are you sure about setting a walltime of 8 hrs?

Reply:
yep